docs/upgrade/UPGRADE-v8.1.0-dev.md

```diff
@@ -79,4 +79,10 @@ There you can find links to upgrade notes for other versions too.
         +       arguments:
         +           $useInlineEditation: false
         ```
+- use `CronModuleExecutor` as a service in DIC ([#1314](https://github.com/shopsys/shopsys/pull/1314))
+    - update your code if you are using `CronFacade::TIMEOUT_SECONDS` constant because it will be removed in next major release
+    - These methods are deprecated and will be removed in the next major release:
+        - `runModulesForInstance()` use method `runModules()` instead
```

Comment:
It would make more sense to mention the class as well for clarity:

```suggestion
        - `CronFacade::runModulesForInstance()` use method `runModules()` instead
```